care/emr/api/viewsets/diagnostic_report.py

```diff
@@ -0,0 +1,168 @@
+from datetime import UTC, datetime
+
+from django.db import models
+from django.db.models import Case, CharField, Value, When
+from django_filters import ChoiceFilter, FilterSet, OrderingFilter, UUIDFilter
+from django_filters.rest_framework import DjangoFilterBackend
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework.decorators import action
+from rest_framework.response import Response
+
+from care.emr.api.viewsets.base import EMRModelViewSet
+from care.emr.models.diagnostic_report import DiagnosticReport
+from care.emr.models.observation import Observation
+from care.emr.resources.diagnostic_report.spec import (
+    DiagnosticReportCreateSpec,
+    DiagnosticReportListSpec,
+    DiagnosticReportObservationRequest,
+    DiagnosticReportRetrieveSpec,
+    DiagnosticReportReviewRequest,
+    DiagnosticReportUpdateSpec,
+    DiagnosticReportVerifyRequest,
+    StatusChoices,
+)
+from care.emr.resources.observation.spec import Performer, PerformerType
+
+
+class PhaseChoices(models.TextChoices):
+    IN_PROCESS = "in_process", "In Process"
+    VERIFICATION_REQUIRED = "verification_required", "Verification Required"
+    REVIEW_REQUIRED = "review_required", "Review Required"
+    REVIEWED = "reviewed", "Reviewed"
+
+
+class DiagnosticReportFilters(FilterSet):
+    phase = ChoiceFilter(choices=PhaseChoices.choices, method="filter_phase")
```

Comment:
We will not maintain any choices in care, all verbose values will be stored in the FE, the FE has to send the right values for filtering, remove the annotation as well.